contrib/graphite/blueflood.py

```diff
@@ -268,15 +289,17 @@ def find_nodes(self, query):
         return self.find_nodes_without_submetrics(query)
 
     except Exception as e:
-     print "Exception in Blueflood find_nodes: "
-     print e
+     logger.exception("Exception in Blueflood find_nodes: ")
      exc_info = sys.exc_info()
      tb = traceback.format_exception(*exc_info)
      for line in tb:
-       print(line)
+       logger.debug(line)
```

Comment:
Nit-pick, I'm a noob on this. But reading the python doc, 
```

Logger.exception(msg, *args, **kwargs)

    Logs a message with level ERROR on this logger. The arguments are interpreted as for debug(), except that any passed exc_info is not inspected. Exception info is always added to the logging message. This method should only be called from an exception handler.
```
It seems like exception info is always added? So do we still need to loop over the lines and print them as logger.debug() ?